test/fixtures/terminal/index.js

```diff
@@ -1,32 +1,16 @@
 /* global Pear */
-const { config,versions } = Pear
-const [grn, rst, dim] = ['\x1b[32m', '\x1b[0m', '\x1b[2m']
-const v = ({ key, length, fork }) => `v${fork}.${length}.${(key += '').length <= 12 ? key : key.slice(0, 12) + '…'}`
-const { app, platform } = await versions()
+const { teardown } = Pear
+import nodeInspector from 'bare-inspector'
+import { Inspector } from 'pear-inspect'
+import Pipe from 'bare-pipe'
 
-const debug = { ready: false, update: false }
+const stdout = new Pipe(1)
+stdout.unref()
 
-const argv = global.Bare?.argv || global.process.argv
-const debarg = argv.find(arg => arg.startsWith('--debug='))
+const inspector = new Inspector({ inspector: nodeInspector})
+const inspectorKey = await inspector.enable()
 
-if (debarg) {
-  const opts = debarg.split('=')[1].split(',')
-  debug.ready = opts.includes('ready')
-  debug.update = opts.includes('update')
-}
+stdout.write(`[ inspect ] key: ${inspectorKey.toString('hex')}\n`)
+stdout.write('[ inspect ] ready\n')
```

Comment:
line isn't needed - ready is implied by having the key 

```
console.log({ key: inspectorKey.toString('hex') })
```

would be fine, no need for bare-pipe then

Reply:
ready line removed